mono/metadata/mono-perfcounters.c

```diff
@@ -741,27 +744,22 @@ instance_search (SharedHeader *header, void *data)
 }
 
 static SharedInstance*
-find_custom_instance (SharedCategory* cat, char *name)
+find_custom_instance (SharedCategory* cat, const char *name)
 {
-	InstanceSearch search;
+	InstanceSearch search = { 0 };
```

Comment:
Line damage. Same for https://github.com/mono/mono/pull/6355/files#diff-fed0ac23b14fd6ea2f7d6974152c702bL750 and https://github.com/mono/mono/pull/6355/files#diff-fed0ac23b14fd6ea2f7d6974152c702bL751